### PR TITLE
add pagination for showing events table

### PR DESCRIPTION
**Pull request by mmonzo-eb**
I created a pagination for showing the table, which allows you to navegate to the first and last pages. It also shows the current page and the 2 previous and next pages, if they exist.

### CODE REVIEW

File: templates/event_list.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
 {% load bootstrap4 %}
 {% bootstrap_css %}
 
+{% load el_pagination_tags %}
+
 <h1>Events</h1>
-<div class="container-fluid">
+
+<div class="container-fluid" style="width:80%">
```
Comment (walterbucolo-eb) [inline templates/event_list.html:8]: You should create a file to put styles. Then you can import this file in the html file.  check this https://docs.djangoproject.com/en/3.0/intro/tutorial06/

File: templates/pagination.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    </li>
+                {% endif %}
+            {% endif %}
+        {% endfor %}
+        
+        <li class="page-item">
+            <a class="page-link" href="{{ pages.last.path }}">Last</a>
+        </li>
+    </ul>
+</nav>
```
Comment (walterbucolo-eb) [inline templates/pagination.html:27]: Newline

File: templates/event_list.html (hunk 1 above)
Comment (mmonzo-eb) [reply templates/event_list.html]: Thank you for the observation. I've just fixed it

File: templates/pagination.html (hunk 2 above)
Comment (mmonzo-eb) [reply templates/pagination.html]: fixed

File: static/search_events/style.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.width-90 {
+    width: 90%;
+}
```
Comment (walterbucolo-eb) [inline static/search_events/style.css:3]: newline

Comment (mmonzo-eb) [reply static/search_events/style.css]: Thanks, fixed! I always forget to add the final blank line

File: static/search_events/style.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.width-90 {
```
Comment (walterbucolo-eb) [inline static/search_events/style.css:1]: NIT In the future, you can choose a more specific name to this component, maybe more related to the table. For example, you can name this as `.table {...}` and put basic attributes, now only width: 90%, but then, you can more attributes if you want.

**Pull request merged**